From e7d219ebe83e2b8a2313b1557cd1442d3d187860 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Sun, 22 Jan 2023 21:36:46 +0000
Subject: [PATCH 51/58] Docs/hwmon/index: Add missing SPDX License Identifier

Add missing SPDX License Identifier for hwmon documentation index file.

Signed-off-by: SeongJae Park <sj@kernel.org>
Reviewed-by: Mauro Carvalho Chehab <mchehab@kernel.org>
Link: https://lore.kernel.org/r/20230122213650.187710-5-sj@kernel.org
Signed-off-by: Guenter Roeck <linux@roeck-us.net>
---
 Documentation/hwmon/index.rst | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/hwmon/index.rst b/Documentation/hwmon/index.rst
index 95f5459ed..96ce91b7d 100644
--- a/Documentation/hwmon/index.rst
+++ b/Documentation/hwmon/index.rst
@@ -1,3 +1,5 @@
+.. SPDX-License-Identifier: GPL-2.0
+
 =========================
 Linux Hardware Monitoring
 =========================
-- 
2.39.2.501.gd9d677b2d8

